addons/bestja_volunteer/models.py

```diff
@@ -126,48 +135,63 @@ def sync_group(self, group, domain):
         })
 
     def check_password_rules(self, password):
```

Comment:
That’s why I added the “new” prefix in my proposition. :P Another one: `password_is_safe`, if it is not provocative. ;)